bundles/net.tourbook/src/net/tourbook/ui/TableColumnFactory.java

```diff
@@ -4697,6 +4700,25 @@ public TableColumnDefinition createColumn(final ColumnManager columnManager,
          }
       };
 
+      NUTRITION_NUM_PRODUCTS = new TableColumnFactory() {
+         @Override
+         public TableColumnDefinition createColumn(final ColumnManager columnManager,
+                                                   final PixelConverter pixelConverter) {
+
+            final TableColumnDefinition colDef = new TableColumnDefinition(columnManager, NUTRITION_NUM_PRODUCTS_ID, SWT.TRAIL);
+
+            colDef.setColumnCategory(           Messages.ColumnFactory_Category_Nutrition);
```

Comment:
The columns are sorted by category and name, the nutrition column is just within the tour category columns

Reply:
Moved it after the `Motion` category